hcloud/core/client.py

```diff
@@ -66,17 +43,14 @@ def _get_all(
 
     def get_all(self, *args, **kwargs):
```

Comment:
Now that `get_all` is virtually the same as `_get_all`, do you think we can combine them and get rid of `_get_all`? Maybe stuff for a follow up PR.

Reply:
Hmmm, the `get_all` and `_get_all` methods do differ, one takes a `get_list` function as argument.

If a child client does not implement the `get_all` method itself, we would fall back to this one, so we probably want to keep this around.

Reply:
Right now `_get_all` is just called by `ClientEntityBase.get_all`. So we can just inline the implementation into `get_all`. 

All clients call `super.get_all(args)`, which would continue to work. I can just prepare a PR with my suggestion after this one merges :)